src/libp2p_transport_tcp.erl

```diff
@@ -31,6 +31,23 @@
 
 -type opt() :: {listen, [listen_opt()]}.
 
+%% list of non-publicly routable IP address blocks in CIDR notation: {IP, bits in mask}
+%% exclude 0.0.0.0/8 as this is required for the relay test suite
+-define(BOGONS, [{{10, 0, 0, 0}, 8},
```

Comment:
nit: With lists like this,  I think a comment with a link to the source you pulled it from is useful.

Reply:
Thanks. Will add the source.